package.json

```diff
@@ -101,7 +101,7 @@
 		"prettier-plugin-sh": "0.18.0",
 		"release-it": "19.0.1",
 		"sentences-per-line": "0.3.0",
-		"tsup": "8.5.0",
+		"tsdown": "^0.13.3",
```

Comment:
I believe the Renovate config will switch this to `0.13.3` quickly after merge. Not an issue as it'll be automatic.